src/screens/AddScreen.js

```diff
@@ -28,12 +30,14 @@ export default function AddScreen({ navigation }) {
     location: { latitude: '', longitude: '' },
     planted: null,
     comments: [],
+    region: null,
   });
   const [newLocation, setNewLocation] = useState({
     latitude: DEFAULT_LOCATION.latitude,
     longitude: DEFAULT_LOCATION.longitude,
   });
   const [checked, setChecked] = React.useState(false);
```

Comment:
im pretty sure you can remove "React" from in front of useState, please do this for all times we use "React" since we import it at the top anyway